### PR TITLE
DOC-1941: Copy edit changes to changelog.md 6.4.1

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-1941: Copy edit changes to changelog.md 6.4.1.

Changes:
* `Hotfix` copy edit changes to changelog.md 6.4.1.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.4.1-release-notes.adoc [hunk 1]
```diff
@@ -93,7 +93,7 @@ In {productname} 6.4.1, the Smart Typography plugin no longer creates a space be
 === Added functionality to remove spaces between exclamation marks
 //#TINY-9398
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/6.4.1-release-notes.adoc:94]:
```suggestion
```

File: modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -17,13 +17,13 @@ NOTE: This is the {productname} Community version changelog. For information abo
...
-* New `isEditable` API to `editor.dom` that returns true or false if the specified node is editable.
-* New `setText` and `setIcon` methods added to menu button and toolbar button API.
+* New `editor.selection.isEditable` API method that returns whether the current selection is editable.
+* New `editor.dom.isEditable` API method that returns whether the specified node is editable.
+* New `setText` and `setIcon` methods added to menu button and toolbar button APIs.
 * New `highlight_on_focus` option which enables highlighting the content area on focus.
 * New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons.
-* Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus.
-* New `font_size_input_default_unit` option allows entry of numbers without a unit in `fontsizeinput`. They are then parsed as the set unit. If `font_size_input_default_unit` is not set the default is `pt`.
+* Added `skip_focus` check to the ToggleToolbarDrawer command.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:25]: No idea what `skip_focus` is

File: modules/ROOT/pages/changelog.adoc [hunk 3]
```diff
@@ -17,13 +17,13 @@ NOTE: This is the {productname} Community version changelog. For information abo
...
+* New `font_size_input_default_unit` option allows to use of numbers without a unit in `fontsizeinput` and have them parsed with the default unit, if it is not defined the default is `pt`.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:26]:
```suggestion
* New `font_size_input_default_unit` option allows to use of numbers without a unit in `fontsizeinput` and have them parsed with the default unit, if it is not defined.  The default unit is `pt`.
```

File: modules/ROOT/pages/changelog.adoc [hunk 4]
```diff
@@ -17,13 +17,13 @@ NOTE: This is the {productname} Community version changelog. For information abo
...
 * New `group` and `togglebutton` in view.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:27]: Are these new buttons or properties?

File: modules/ROOT/pages/changelog.adoc [hunk 5]
```diff
@@ -41,31 +41,31 @@ NOTE: This is the {productname} Community version changelog. For information abo
...
+* Sometimes the editor would finish initializing before the theme would have finished loading.
 * The `searchreplace` modal closed incorrectly when clicking outside of the alert that pops up when no match is found.
 * The text color or background color picker toolbar buttons did not update when the text color or background color was changed using the equivalent commands in the Format menu.
 * The `onSetup` api function would not run when defining custom group toolbar button.
 * The foreground and background menu icons would not properly update to display the last used color.
-* Added new `setIconFill` function to `NestedMenuItemInstanceApi`.
-* Pasting links to text would sometimes not generate the correct undo stack in Safari.
-* Toolbar split buttons in `advlist` plugin now show the correct state when the cursor is in a checklist.
+* Added new `setIconFill` function to `editor.ui.addNestedMenuItem`.
+* Pasting links to text would sometimes not generate the correct undo stack on Safari.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:60]:
```suggestion
* Pasting links to text would sometimes cause the incorrect undo state on Safari.
```

????

File: modules/ROOT/pages/changelog.adoc [hunk 6]
```diff
@@ -75,7 +75,7 @@ NOTE: This is the {productname} Community version changelog. For information abo
 * Textareas with scrollbars in dialogs would not render rounded corners correctly on some browsers.
 * It was possible to open links inside the editor if the editor root was an element with a `contenteditable="false"` attribute.
 * Inline boundary was rendered for boundary elements that had a `contenteditable="false"` attribute.
-* Clicking on a disabled split button will no longer call the `onAction` callback.
+* Clicking on a disabled split button would call the onAction callback.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:78]:
```suggestion
* Clicking on a disabled split button would call the `onAction` callback.
```

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/changelog.adoc (hunk 2 above)
Comment (MitchCTiny) [reply modules/ROOT/pages/changelog.adoc:25]: its an optional argument for certain commands which involve UI elements, when false or null the editor will be focused after the command is run, but when set to true this focus step will be skipped.

**Review by MitchCTiny — APPROVED** [review]

File: modules/ROOT/pages/changelog.adoc (hunk 3 above)
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:26]: Closing without commiting suggestion as discussed.

File: modules/ROOT/pages/changelog.adoc (hunk 5 above)
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:60]: Closing without committing suggestion as discussed.

File: modules/ROOT/pages/changelog.adoc (hunk 4 above)
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:27]: Closing without committing suggestion as discussed.

**Pull request merged**